Gitlab.follow() throws Json Exception

Seems like Gitlab.follow(...) throws an exception:

```java
2021-04-28 16:15:06.511 DEBUG 8831 --- [scheduling-1] com.selfxdsd.core.FollowProjectOwner     : Following PO...
2021-04-28 16:15:07.352 ERROR 8831 --- [scheduling-1] o.s.s.s.TaskUtils$LoggingErrorHandler    : Unexpected error occurred in scheduled task

javax.json.stream.JsonParsingException: JsonParser#getObject() or JsonParser#getObjectStream() is valid only for START_OBJECT parser state. But current parser state is START_ARRAY
	at org.glassfish.json.JsonReaderImpl.readObject(JsonReaderImpl.java:114) ~[javax.json-1.1.4.jar!/:1.1.4]
	at com.selfxdsd.core.JsonResources$JsonResponse.asJsonObject(JsonResources.java:351) ~[self-core-impl-0.0.77.jar!/:na]
	at com.selfxdsd.core.Gitlab.findUserId(Gitlab.java:186) ~[self-core-impl-0.0.77.jar!/:na]
	at com.selfxdsd.core.Gitlab.follow(Gitlab.java:149) ~[self-core-impl-0.0.77.jar!/:na]
	at com.selfxdsd.core.FollowProjectOwner.accept(FollowProjectOwner.java:81) ~[self-core-impl-0.0.77.jar!/:na]
	at com.selfxdsd.selfpm.AcceptInvitations.acceptInvitations(AcceptInvitations.java:86) ~[classes!/:0.0.8-SNAPSHOT]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
	at org.springframework.scheduling.support.ScheduledMethodRunnable.run(ScheduledMethodRunnable.java:84) ~[spring-context-5.2.6.RELEASE.jar!/:5.2.6.RELEASE]
	at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlingRunnable.java:54) ~[spring-context-5.2.6.RELEASE.jar!/:5.2.6.RELEASE]
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[na:na]
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) ~[na:na]
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: JsonParser#getObject() or JsonParser#getObjectStream() is valid only for START_OBJECT parser state. But current parser state is START_ARRAY
	at org.glassfish.json.JsonParserImpl.getObject(JsonParserImpl.java:171) ~[javax.json-1.1.4.jar!/:1.1.4]
	at org.glassfish.json.JsonReaderImpl.readObject(JsonReaderImpl.java:112) ~[javax.json-1.1.4.jar!/:1.1.4]
	... 17 common frames omitted
```

--- a/self-core-impl/src/main/java/com/selfxdsd/core/Gitlab.java
+++ b/self-core-impl/src/main/java/com/selfxdsd/core/Gitlab.java
@@ -25,6 +25,7 @@ package com.selfxdsd.core;
 import com.selfxdsd.api.*;
 import com.selfxdsd.api.storage.Storage;
 
+import javax.json.JsonArray;
 import javax.json.JsonValue;
 import java.net.HttpURLConnection;
 import java.net.URI;
@@ -183,7 +184,12 @@ public final class Gitlab implements Provider {
         );
         final int id;
         if (response.statusCode() == HttpURLConnection.HTTP_OK) {
-            id = response.asJsonObject().getInt("id");
+            final JsonArray result = response.asJsonArray();
+            if (!result.isEmpty()) {
+                id = result.get(0).asJsonObject().getInt("id");
+            } else {
+                id = -1;
+            }
         } else {
             id = -1;
         }

--- a/self-core-impl/src/test/java/com/selfxdsd/core/GitlabITCase.java
+++ b/self-core-impl/src/test/java/com/selfxdsd/core/GitlabITCase.java
@@ -188,14 +188,19 @@ public final class GitlabITCase {
                     .endsWith("/users?username=john")) {
                     mock = new MockResource(
                         HttpURLConnection.HTTP_OK,
-                        Json.createObjectBuilder()
-                            .add("id", 1)
+                        Json.createArrayBuilder()
+                            .add(
+                                Json.createObjectBuilder()
+                                    .add("id", 1)
+                                    .build()
+                            )
                             .build()
+
                     );
                 } else {
                     mock = new MockResource(
-                        HttpURLConnection.HTTP_NOT_FOUND,
-                        JsonValue.NULL
+                        HttpURLConnection.HTTP_OK,
+                        JsonValue.EMPTY_JSON_ARRAY
                     );
                 }
             } else if ("POST".equals(method)) {

